File: lat3_c++/Processor.cpp
#include<iostream>
#include<string>
using namespace std;

class Processor
{
private:
    string name;
    int price;
    
public:
    //constructor
    Processor(){}
    Processor(string name, int price){
        this->name = name;
        this->price = price;
    }

    //setter & getter 
    void setName(string name)
    {
        this->name = name;
    }

    string getName()
    {
        return this->name;
    }

    void setPrice(int price)
    {
        this->price = price;
    }

    int getPrice()
    {
        return this->price;
    }

    void outputProcessor()
    {
        cout << "Processor name     : " << this->getName() << endl;
        cout << "Processor Price    : " << this->getPrice() << endl;
    }

    // destructor
    ~Processor() {}
};   

File: lat3_python/Processor.py
class Processor():

    __name = ""
    __price = ""

    def __init__(self, name = "", price = ""):
        self.__name = name
        self.__price = price

    def setName(self, name):
        self.__name = name

    def setPrice(self, price):
        self.__price = price

    def getName(self):
        return self.__name

    def getPrice(self):
        return self.__price

    def printProcessor(self):
        print("Processor Name   : ", str(self.getName()))
        print("Processor Price  : ", str(self.getPrice()))